.github/workflows/build.yml

```diff
@@ -29,37 +29,27 @@ jobs:
         run: |
           echo "::set-env name=PY_VERSION::"\
           "$(python -c "import platform;print(platform.python_version())")"
-      - name: Cache pip test requirements
-        uses: actions/cache@v1
-        with:
-          path: ${{ env.PIP_CACHE_DIR }}
-          key: "${{ runner.os }}-pip-test-py${{ env.PY_VERSION }}-\
-            ${{ hashFiles('**/requirements-test.txt') }}"
-          restore-keys: |
-            ${{ runner.os }}-pip-test-py${{ env.PY_VERSION }}-
-            ${{ runner.os }}-pip-test-
-            ${{ runner.os }}-pip-
-      - name: Cache pre-commit hooks
-        uses: actions/cache@v1
-        with:
-          path: ${{ env.PRE_COMMIT_CACHE_DIR }}
-          key: "${{ runner.os }}-pre-commit-py${{ env.PY_VERSION }}-\
-            ${{ hashFiles('**/.pre-commit-config.yaml') }}"
-      - name: Cache curl downloads
-        uses: actions/cache@v1
-        with:
-          path: ${{ env.CURL_CACHE_DIR }}
-          key: "${{ runner.os }}-curl-\
-            terraform-${{ env.TERRAFORM_VERSION }}"
       - name: Lookup go cache directory
         id: go-cache
         run: |
           echo "::set-output name=dir::$(go env GOCACHE)"
-      - name: Cache go
-        uses: actions/cache@v1
+      - name: Cache linting environments
+        uses: actions/cache@v2
         with:
-          path: ${{ steps.go-cache.outputs.dir }}
-          key: "${{ runner.os }}-go"
+          path: |
+            ${{ env.PIP_CACHE_DIR }}
+            ${{ env.PRE_COMMIT_CACHE_DIR }}
+            ${{ env.CURL_CACHE_DIR }}
+            ${{ steps.go-cache.outputs.dir }}
+          key: "lint-${{ runner.os }}-\
```

Comment:
`go` is not mentioned in the key.  Should it be?

Reply:
The previous key didn't use any versioning so I left it off. I can add it back if desired, I'd just need to get the version so the cache gets updated if we change the Go version being installed.

Reply:
If we are including the Python version, then we probably ought to include the Go version as well, right?

Reply:
I think it makes sense to include it here, and we should have done it in cisagov/skeleton-packer#42 as well.  [cisagov/skeleton-packer](https://github.com/skeleton-packer) will need a new PR.

What say you, @felddy and @hillaryj?

Reply:
We opted to add the go version to the cache key in cisagov/skeleton-packer#43, so I did it here as well in commit 8c2277e.  

While making that change I noticed that the `.terraform` directory was never added to the list of cached directories, so I went ahead and made that change in commit 5855358.